web/src/components/BasicPage/BasicPage.css

```diff
@@ -1,4 +1,6 @@
 .body-lower-half {
```

Comment:
I wouldn't style basic page if I were you. The point of basic page is to prevent  repeating navbar and footer in every new page we make.  I think this is adding the lower purple picture to every page (see about page)

Reply:
yes ! I have already taken this away

Reply:
@alodahl when you get a chance could you take a look at the changes i made? When I set the top of the projects component the MazeBottom isn't scooting up